drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
Well, there are several other examples which are not possible to describe using this simple language. I wanted to keep this commit small. Of course I'm planning to improve our syntax soon. Added a reference to i#1948 where we have a bit more details about this.